noir-projects/noir-protocol-circuits/crates/rollup-lib/src/base/private_base_rollup.nr

```diff
@@ -1004,24 +1007,24 @@ mod tests {
         }
         offset += total_private_logs_len;
         // cc logs
-        let total_contract_class_logs_len = NUM_CC_LOGS * (cc_log.len() + 1);
+        let total_contract_class_logs_len = NUM_CC_LOGS * (cc_log.len() + 2);
```

Comment:
I just opened an issue for the pattern in this file https://github.com/AztecProtocol/aztec-packages/issues/12217

Reply:
Just FYI I think this behaviour is mostly in tests. We need it for the blob because we don't want to concat arrays, we want to fill an array with certain elements conditionally.